From db6e0dd4d2f8eaeec287219f1912a060f250f714 Mon Sep 17 00:00:00 2001
From: Moez Bouhlel <bmoez.j@gmail.com>
Date: Wed, 14 Jan 2015 14:33:32 +0000
Subject: [PATCH] fix zsh completion typo

json-see => json-sse

(cherry picked from commit d31f44e2818eb00213ca718e05b9bc624e7bfe3f)
---
 shell-completion/zsh/_sd_outputmodes | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell-completion/zsh/_sd_outputmodes b/shell-completion/zsh/_sd_outputmodes
index 2ce84a79a1..3836f79b73 100644
--- a/shell-completion/zsh/_sd_outputmodes
+++ b/shell-completion/zsh/_sd_outputmodes
@@ -1,5 +1,5 @@
 #autoload
 
 local -a _output_opts
-_output_opts=(short short-iso short-precise short-monotonic verbose export json json-pretty json-see cat)
+_output_opts=(short short-iso short-precise short-monotonic verbose export json json-pretty json-sse cat)
 _describe -t output 'output mode' _output_opts || compadd "$@"
